function-aws/src/main/java/io/micronaut/function/aws/MicronautRequestHandler.java

```diff
@@ -143,7 +147,14 @@ public final O handleRequest(I input, Context context) {
         if (!inputType.isInstance(input)) {
             input = convertInput(input);
         }
-        return this.execute(input);
+        try {
+            O output = this.execute(input);
+            resolveAfterExecutionPublisher().publishEvent(AfterExecutionEvent.success(output));
```

Comment:
does it make sense to include the `Context` in the event?

Reply:
I'm not sure how much it's worth for the`success` but it could be definitely useful for the`failure`. I will add it.